rust-sdk/whirlpool/Cargo.toml

```diff
@@ -28,6 +28,7 @@ orca_whirlpools_client = { path = "../client", features = ["core-types"] }
 bincode = { version = "^1.3" }
 serde = { version = "^1.0" }
 serde_json = { version = "^1.0" }
+tokio = { version = "1.0", features = ["sync"] }
```

Comment:
Is this needed as a normal dep or can it be a dev-dep?

Reply:
Good point, need to delete